solc/CommandLineInterface.cpp

```diff
@@ -653,7 +654,8 @@ void CommandLineInterface::processInput()
 		assemble(m_options.assembly.inputLanguage, m_options.assembly.targetMachine);
 		break;
 	case InputMode::Linker:
-		link();
+		solAssert(m_options.input.mode == InputMode::Linker, "");
```

Comment:
We have recently made the description optional in asserts:
```suggestion
		solAssert(m_options.input.mode == InputMode::Linker);
```